dm/ctl/master/check_task.go

```diff
@@ -44,7 +44,7 @@ func checkTaskFunc(cmd *cobra.Command, _ []string) {
 	}
 	content, err := common.GetFileContent(cmd.Flags().Arg(0))
 	if err != nil {
-		common.PrintLines("get file content error:\n%v", errors.ErrorStack(err))
+		common.PrintLines("get file content error:\n%v", terror.Message(err))
```

Comment:
How about returning `Error` directly? for `terror`, it will contain useful `code`, `class`, etc.